src/Components/Search/SearchBar.tsx

```diff
@@ -94,7 +94,7 @@ const SuggestionContainer = ({ children, containerProps, preview }) => {
             {children}
           </Flex>
         </SuggestionsWrapper>
-        <Box width={["0px", "240px", "240px", "450px"]} pl={[0, 3]} py={[0, 2]}>
+        <Box width={["0px", "240px", "240px", "450px"]} px={[0, 3]} py={[0, 2]}>
```

Comment:
This, and [line 36 in `PreviewGridItem`](https://github.com/artsy/reaction/pull/2082/files#diff-1ae621323dbe87da0a101e12f5c58328L36), are the main thing I wanted to accomplish.